src/containers/Jokes/Jokes.js

```diff
@@ -0,0 +1,204 @@
+import React, { Component } from "react";
+import Joke from "../../components/Joke/Joke";
+import "./jokes.css";
+
+class Jokes extends Component {
+  state = {
+    jokes: [
```

Comment:
Testing data should be moved to separate file in same folder, after this will be connected to real API, the file can be deleted.